ci/prow/Makefile

```diff
@@ -33,8 +33,13 @@ config:
 
 update-config:
 	$(SET_CONTEXT)
+	$(eval TMP_YAML_CONFIG := $(shell mktemp))
+	kubectl get configmap config -o jsonpath="{.data}" 2>/dev/null >"${TMP_YAML_CONFIG}"
+	sed -i -e "s/((COMMIT_HASH_TOKEN))/$(shell git rev-parse HEAD)/" "config.yaml"
 	kubectl create configmap config --from-file=config.yaml=config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+	git checkout config.yaml
 	$(UNSET_CONTEXT)
+	@echo "Old config file saved at: ${TMP_YAML_CONFIG}"
```

Comment:
What about saving it to a GCS bucket with a timestamp, instead of a local file on a workstation?

Reply:
good idea, done